src/adapters/http/index.ts

```diff
@@ -0,0 +1,109 @@
+import Adapter from '../../lib/adapter.js';
+import GleeMessage from '../../lib/message.js';
+import http from 'http';
+import { validateData } from '../../lib/util.js';
+import GleeError from '../../errors/glee-error.js';
+import * as url from 'url';
+
+class HttpAdapter extends Adapter {
+    private res: any
+
+    name(): string {
+        return 'HTTP server'
+    }
+
+    async connect(): Promise<this> {
+        return this._connect();
+    }
+
+    async send(message: GleeMessage): Promise<void> {
+        console.log('control reached send');
+        return this._send(message);
+    }
+
+    _connect(): Promise<this> {
+        return new Promise((resolve, reject) => {
+            const serverUrl = new URL(this.serverUrlExpanded);
+            const httpServer = this.glee.options?.websocket?.httpServer || http.createServer();
+            const asyncapiServerPort = serverUrl.port || 80;
+            const optionsPort = this.glee.options?.websocket?.port;
+            const port = optionsPort || asyncapiServerPort;
+
+            httpServer.on('request', (req, res) => {
+                this.res = res;
```

Comment:
Have you tested what happens when we receive 2 or more requests in a row? Doesn't `this.res` get overwritten?